File: frontend/src/app/_models/dto/UpdateEventDTO.ts
import {Event} from "../event";
import {Location} from "../../_models/location";


export class UpdateEventDTO {
  event: Event;
  newPeople: string[];
  removedPeople: string[];
  frequencyNumber: number;
  frequencyPeriod: string;
  priority: string;
  startTimeNotification: string;
  location: Location;
}

File: backend/src/main/java/com/ncgroup2/eventmanager/dto/UpdateEventDTO.java
package com.ncgroup2.eventmanager.dto;

import com.ncgroup2.eventmanager.entity.Event;
import com.ncgroup2.eventmanager.entity.Location;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class UpdateEventDTO {
    private Event event;
    private List<String> newPeople;
    private List<String> removedPeople;
    private Long frequencyNumber;
    private String frequencyPeriod;
    private String priority;
    private Location location;

}
